### PR TITLE
Feat: import GeoDataFrames

### PR DESCRIPTION
**Pull request by svenruf**
added trackintel/io/from_geopandas.py: Created functions that warp pd.rename to import GeoDataFrames into the trackintel model
added tests for these functions in tests/io/test_io.py and datasets for these tests in tests/data (geojson files)

the commits regarding the test_pypi file can be ignored as they were deleted. Unfortunately I could not get rid of these commits (also the hard reset did not remove them...) 

the issue with the locations (multiple geometry) is related to the issues [867](https://github.com/geopandas/geopandas/issues/867) / [1275](https://github.com/geopandas/geopandas/issues/1275) in Geopandas

### CODE REVIEW

File: tests/io/test_io.py [hunk 1]
```diff
@@ -79,3 +123,14 @@ def test_trips_from_to_postgis(self):
...
 
+    def test_trips_from_gpd(self): 
+        csv_file = os.path.join('tests','data','trips.csv')
+        df = pd.read_csv(csv_file, sep=';')
+        trps = ti.io.from_geopandas.trips_from_gpd(df)
+        trps_csv = ti.read_trips_csv(csv_file,sep=';')
+        trps_csv['started_at'] = trps_csv['started_at'].apply(lambda d: d.isoformat().replace(' ', 'T'))
+        trps_csv['finished_at'] = trps_csv['finished_at'].apply(lambda d: d.isoformat().replace(' ', 'T'))
+        trps_csv['started_at'] = trps_csv['started_at'].apply(lambda d: d.replace('+00:00', 'Z'))
+        trps_csv['finished_at'] = trps_csv['finished_at'].apply(lambda d: d.replace('+00:00', 'Z'))
```
Comment (hong2223) [inline tests/io/test_io.py:134]: same as above, here it is a bit more confusing..

File: trackintel/io/from_geopandas.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    
+    lcs = gdf.rename(columns=columns)
+    lcs = lcs.set_geometry('center')
+    
+    assert lcs.as_locations
+    
+
+    return lcs
+
+def tours_from_gpd(gdf, user_id='user_id',started_at='started_at', finished_at='finished_at', origin_destination_location_id='origin_destination_location_id', journey='journey', mapper={}):
```
Comment (hong2223) [inline trackintel/io/from_geopandas.py:170]: for this function, there are no tests available? Maybe add an issue saying that test for tours are missing in both read_csv and from_geopandas

**Review by hong2223 — COMMENTED** [review]
Nice job! Before we can merge, maybe we need to add some more descriptions to the functions (see the specific comments).
We are also trying to organize the file structures in the 'test' folder. The general idea is to imitate the file structure of the corresponding modules and to create a class for each function. You can see the file 'test_positionfixes.py' and issue #23 as examples. 
So here, maybe create a separate test file named `test_from_geopandas.py`.

**Comment by svenruf** [comment]
Updated the PR with the points discussed in our meeting:
- comments on the hardcoded datetime modifications
- cleaned up the structure of the tests
Anything to add?

**Comment by hong2223** [comment]
Nice job!

**Pull request merged**